mkt/webapps/indexers.py

```diff
@@ -258,9 +259,9 @@ def extract_document(cls, pk=None, obj=None):
             status = None
 
         attrs = ('app_slug', 'bayesian_rating', 'created', 'default_locale',
-                 'icon_hash', 'id', 'is_disabled', 'is_offline', 'file_size',
-                 'last_updated', 'modified', 'premium_type', 'status',
-                 'uses_flash')
+                 'guid', 'icon_hash', 'id', 'is_disabled', 'is_offline',
```

Comment:
Can you also add `guid` to `ESAppSerializer.fake_object(), in the first `self._attach_fields()` call ` ? Otherwise it won't be available to the serializer ; Not a big deal since we don't use it, but we might ?